event/src/deterministic_event.rs

```diff
@@ -0,0 +1,25 @@
+use crate::args::UnsignedEvent;
+use anyhow::Result;
+use ceramic_core::DagCborEncoded;
+use serde::Serialize;
+use std::fmt::Formatter;
+
+/// A deterministic event, where it will always hash the same way
+pub struct DeterministicEvent {
```

Comment:
Maybe we should call this "DeterministicInitEvent"? DataEvents are never "deterministic"